tests/osgi-tests/src/test/java/org/wso2/carbon/osgi/utils/manifest/ManifestElementOSGiTest.java

```diff
@@ -56,29 +58,22 @@
 
     @Configuration
     public Option[] createConfiguration() {
-
-        List<Option> optionList = new ArrayList<>();
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.deployer.mgt").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.order.resolver").groupId("org.wso2.carbon")
-                .versionAsInProject());
-
-        optionList = OSGiTestConfigurationUtils.getConfiguration(optionList, null);
-        return optionList.toArray(new Option[optionList.size()]);
+        return new Option[] { copyDropinsBundle(
+                maven().artifactId("org.wso2.carbon.sample.deployer.mgt").groupId("org.wso2.carbon")
+                        .versionAsInProject()), copyDropinsBundle(
+                maven().artifactId("org.wso2.carbon.sample.order.resolver").groupId("org.wso2.carbon")
+                        .versionAsInProject()) };
     }
 
     @Test
     public void testParseHeader() {
-        Bundle carbonCoreBundle = Arrays.asList(bundleContext.getBundles())
-                .stream()
-                .filter(b -> b.getSymbolicName().equals("org.wso2.carbon.sample.deployer.mgt"))
-                .findFirst()
-                .get();
+        Bundle carbonCoreBundle = Arrays.asList(bundleContext.getBundles()).stream()
```

Comment:
Having different stream operations in separate line improves the readability. 
It's ok to have .stream() on the same line. But if you move the others to different lines it will improve the readability